src/state/routing/types.ts

```diff
@@ -2,13 +2,21 @@ import { Trade } from '@uniswap/router-sdk'
 import { Currency, Token, TradeType } from '@uniswap/sdk-core'
 
 export enum TradeState {
-  LOADING,
   INVALID,
+  LOADING,
```

Comment:
is loading def an invalid trade state? feels like this could cause bugs down the line